### DISCUSSION TITLE
Friendly organizations and foundations

### DISCUSSION THREAD

**Discussion by mkorbi** [cncf/tag-env-sustainability#16] [General]
Yet another list/collection:

which other organizations are out there with whom we can work and collaborate?
This would be also a required input for growing our network and the community. # better together

Topic related:
- [Green Software Foundation](https://greensoftware.foundation/)
- [Open Compute Project](https://www.opencompute.org/)

Extended Community (having similar thoughts or workstreams)
- [GitOps WG](https://github.com/cncf/tag-app-delivery/tree/main/gitops-wg)
- CNCF - like for the cloud surveys etc.

End User or Adopter
- Google
- AWS
- Azure

I believe this is also a never-ending work. However, we should also use this to document how we work together.

**Comment by lasomethingsomething** [comment]
@mkorbi I'd be interested in helping with outreach efforts to such groups.

VMW also has a sustainability program—would add to the list. This blog post gives some info: https://octo.vmware.com/advancing-sustainability-innovation-at-vmware/

Meantime, see also:

- [ClimateAction.tech](http://tedx.climateaction.tech/)

**Reply by mkorbi** [reply]
I think what could be great is at the point we feel ready, to reach out to them at their next meeting and say hi. @wojtekidd was also interested to build the bridges, so maybe we can have here a good tandem to reach out.

**Comment by mkorbi** [comment]
Maybe a little bit farther "away" but overall very interesting to get in touch with
* https://www.footprintnetwork.org/
* https://www.overshootday.org/solutions/

**Comment by leonardpahlke** [comment]
FYI, we are also brainstorming some ideas about partners and initiatives in [docs](https://docs.google.com/document/d/1te0RRqgqCZWyXuj2Pwnth4XPVzY2hMIkIIKjmTIw3zI/edit
)

cc @wojtekidd

**Reply by leonardpahlke** [reply]
also ref: https://github.com/cncf/wg-env-sustainability/issues/8

**Reply by leonardpahlke** [reply]
I will create a PR soon to initialise the list :)

**Comment by wojtekidd** [comment]
The more organizations and projects come to our attention, the more I think we would need to focus on not duplicating what they're already done/are doing. That would need a closer look at their activities and how live/stale they are.

**Reply by mkorbi** [reply]
We could create an md listing and "categorizing" them as well as their current projects

**Reply by wojtekidd** [reply]
Just a thought - how about we reach out to them (by email first) and:

-  introduce ourselves
- tell them what initiatives we want to start or issues we want to look into
- ask whether they already have some expertise around these topics
- if they do, we try to to connect more and build on their current experience, consolidate knowledge

WDYT?

**Reply by wojtekidd** [reply]
It would be great if we could solidify the knowledge that's already there under the CNCF flag

**Comment by lasomethingsomething** [comment]
- [Green Web Foundation](https://www.thegreenwebfoundation.org)
- [Green Coding Berlin](https://www.green-coding.org)
- [Open Climate Fix](https://www.openclimatefix.org)

**Reply by wojtekidd** [reply]
Do these ever end? 😛 TBH it's great that there is so much groups already 💚

**Reply by mkorbi** [reply]
Maybe you can open a work doc for this and like here anticipated getting a first draft flowing (forget about md :D )
> We could create an md listing and "categorizing" them as well as their current projects

**Comment by catblade** [comment]
You miss, on end-users, enterprise and telco.  These are not insignificant markets, especially not in China, and comprise the bulk of the K8s clusters there.

**Reply by nikhil-vytla** [reply]
What kinds of approaches would you recommend in terms of reaching out to these markets?

**Reply by catblade** [reply]
RH is one provider of enterprise software, but there are others.  We need to basically not necessarily reach out directly, but leave space for them.  They are all fairly active in following the community, and if we build things that are useful to them, they will show up.

**Reply by williamcaban** [reply]
Maybe not a well known fact: Telecom operators account for [2-3%](https://inform.tmforum.org/insights/2021/08/can-the-telecoms-industry-power-down-its-impact-on-the-environment/) of the TOTAL power consumption of HUMANITY. Even with the optimizations of 5G, the overall 5G networks consume many times more than previous services, so this ~3% is expected to increase significantly. The adoption of K8s as the preferred platform for 5G and the dependencies of 5G stacks (the CNF apps) in other CNCF projects, if as part of CNCF we can properly measure and provide mechanisms/tools to optimize for a 0.1% of that, that will have a direct and huge impact to the overall sustainability of the industry (everyone uses a communication provider) and the planet.

In terms of reaching out to these markets, I can help with that.

_W

**Reply by mkorbi** [reply]
@williamcaban do you think it makes sense to start with the WG CNF? 
Within the German/British Telco Market, I would also have a good connection.

**Reply by williamcaban** [reply]
@mkorbi The scope of the WG CNF is very limited wrt these topics and when it comes to Telco there are other standards organizations like O-RAN WG6 & WG4, ETSI Sustainability & Environmental Engineering (EE), 3GPP specification, GSMA Sustainability framework, etc, all of which are working with these topics, have the industry recognition and it aggregates communication providers, platform providers and academia into those groups. I would avoid replicating work if something is already working in another community or standard organization and instead focus on consuming those output and specs working with actual implementations.

**Reply by taylor** [reply]
The CNF WG is about helping the Telecom domain in adopting practices which are aligned with those which have driven implementation in K8s core and majority of CNCF/K8s major projects. That is the cloud native principles which are based on existing methodologies and practices which were seen as beneficial in cloud computing environments and technologies.  

Adopting cloud native practices in telecom application, which is the focus on the CNF WG, is related to the topic of sustainability the same as the rest of K8s (core and applications).  

If its being promoted in other areas of K8s it is very likely to be applicable for Telecom and thus relevant to the CNF WG.

cc: @tomkivlin @electrocucaracha @jeffsaelens

**Reply by tomkivlin** [reply]
Agreed @taylor. Vodafone are already listed in the charter as an interested party, and one of our 3 "purpose pillars" is planet - everything we do including k8s and cloud native engineering has energy optimisation and sustainability at it's core.

**Comment by rossf7** [comment]
Open Compute Project are already here but I wanted to mention the panel on "Does Green Software need Open Hardware?" that they held with CNCF at KubeCon in Valencia in May. https://kccnceu2022.sched.com/event/11767

It was a great discussion but sadly I don't think it was recorded. It might be worth reaching out to the panelists?

**Reply by nieebel** [reply]
https://www.youtube.com/watch?v=E0MdTDtFi4M

**Reply by ng-druid** [reply]
Hello, I'm the founder of [Roll the Cloud](https://rollthecloud.com/) registered [501(c)3 charity](https://www.guidestar.org/profile/88-1926859) our [projects](https://github.com/rollthecloudinc) feature [druid](https://github.com/rollthecloudinc/druid) one of the first CMS platforms to make sustainability part of its [core architectural design](https://github.com/rollthecloudinc/druidcloud/wiki/Sustainable-By-Design) from the beginning.

**Comment by arieljatib** [comment]
The [FinOps Foundation has a SIG](https://www.finops.org/projects/overview/) focused on Sustainability. The efforts, including the WG, is led by @marksbutcher. 

Mark is interested in collaborating on these efforts; he is also the organizer of the [London FinOps Meetup](https://www.meetup.com/finops-uk/) and [spoke on sustainability at the recent FinOps X conference](https://finopsx22.sched.com/speaker/mark2081).

**Comment by Marksbutcher** [comment]
[FinOps Foundation](finops.org) are interested. I lead the sustainability working group in the Foundation

**Comment by mrchrisadams** [comment]
hello folks. I work in the policy working group at the Green Software Foundation. I'm looking for recommendations for an analyst or boutique analyst firm to help us get a landscape analysis together, and map out the different areas of focus these groups have.

The primary goal is helping inform the green software foundations own policy advocacy over the next 24 months or so, but I'm hoping we'll end up with some shareable artefact that might be relevant for other groups in this field as well.

This a funded project, and we're intending to commission it over the summer, to end up with a deliverable that lists existing initiatives and how they might relate to existing anticipated policy changes over the next 24 months. We have an annual drumbeat of the COP process now for example, as well as energy market reform in various regions, along with new requirements on emissions disclosure coming in Europe and the States as examples which are likely to be drivers in this field.

If you know any analysts or firms who

a) would be a in position to deliver this work and 
b) be able to do so with some degree of disinterest and impartiality

I'd really appreciate comments referring to them below.

Thanks!

P.S. In the interests of disclosure, I'm also on the management team of the Green Web Foundation, and I help organise climateAction.tech.

**Reply by mkorbi** [reply]
Hey @mrchrisadams that sounds like very great action, especially if it is professionally funded. I would hope that we can have someone fo the team also joining you, at least to listen and see where it is going and if we can bring some additional value.

In addition to that, I can ask in my network, there is these Leaders for Climate Action initiative where tons of different companies are in. I will check with them!

**Comment by Marksbutcher** [comment]
Would be good to talk about how we can potentially align the work being created by the FinOps Foundation in this area. We’re also building out a series of artefacts for use by FinOps Practitioners.

Thanks

Mark

On 25 Jul 2022, at 10:45, Chris Adams ***@***.***> wrote:

﻿

hello folks. I work in the policy working group at the Green Software Foundation. I'm looking for recommendations for an analyst or boutique analyst firm to help us get a landscape analysis together, and map out the different areas of focus these groups have.

The primary goal is helping inform the green software foundations own policy advocacy over the next 24 months or so, but I'm hoping we'll end up with some shareable artefact that might be relevant for other groups in this field as well.

This a funded project, and we're intending to commission it over the summer, to end up with a deliverable that lists existing initiatives and how they might relate to existing anticipated policy changes over the next 24 months. We have an annual drumbeat of the COP process now for example, as well as energy market reform in various regions, along with new requirements on emissions disclosure coming in Europe and the States as examples which are likely to be drivers in this field.

If you know any analysts or firms who

a) would be a in position to deliver this work and
b) be able to do so with some degree of disinterest and impartiality

I'd really appreciate comments referring to them below.

Thanks!

P.S. In the interests of disclosure, I'm also on the management team of the Green Web Foundation, and I help organise climateAction.tech.

**Comment by arieljatib** [comment]
I came upon [a report from the IEA](https://www.iea.org/reports/electricity-information-overview) (International Energy Agency) on electricity production, consumption and trade. While I am not familiar with the organization, this report provides some statistics and graphs which might prove useful and inform our efforts. 

> Electricity Information provides a comprehensive review of historical and current market trends in the OECD electricity sector. It provides an overview of the world electricity developments covering world electricity and heat production, input fuel mix, supply and consumption, and electricity imports and exports.

> ... It provides comprehensive statistical details on overall energy consumption, economic indicators, electricity and heat production by energy form and plant type, electricity imports and exports, sectoral energy and electricity consumption, as well as prices for electricity and electricity input fuels for each country and regional aggregate.

**Comment by mkorbi** [comment]
tracking item and further discussion TBD here https://github.com/cncf/tag-env-sustainability/issues/26
